bin/get_twinkles_visits.py

```diff
@@ -4,20 +4,57 @@
 file given a fieldID.
 """
 from __future__ import absolute_import
+import os
 import argparse
+import pandas as pd
 from desc.twinkles import get_twinkles_visits
+from desc.twinkles import OpSimOrdering
+from lsst.utils import getPackageDir
 
 parser = argparse.ArgumentParser(description="Write an ascii file of visits derived from an OpSim db file given a fieldID")
 parser.add_argument('opsimDB', help='OpSim database sqlite file')
 parser.add_argument('--fieldID', type=int, default=1427,
                     help='ID number of the desired field')
 parser.add_argument('--outfile', type=str, default=None, help='output file')
+parser.add_argument('--orderObsHistIDsByDesign', type=bool, default=True,
+                    help='order obsHistIDs as decided for Twinkles 3')
+parser.add_argument('--randomForestPickleFileName', type=str, default='RF_pickle.p',
```

Comment:
And I agree about it being run 3 specific. I don't know how to refactor it more generally